src/handlers/handle-guild-message-creation.ts

```diff
@@ -0,0 +1,27 @@
+import type { Message } from 'discord.js';
+import { MessageType } from 'discord.js';
+
+export const handleGuildMessageCreation = async (message: Message) => {
+  if (message.author.bot) {
+    return;
+  }
+
+  if (message.type !== MessageType.Default) {
+    return;
+  }
+
+  const urls = message.content.match(/https?:\/\/\S+/g);
+  if (urls === null) {
+    return;
+  }
+
+  const newContent = message.content.replaceAll(
+    /https?:\/\/(mobile\.)?twitter\.com/g,
+    'https://vxtwitter.com'
+  );
+
+  const newMessage = [`<@${message.author.id}>`, newContent].join('\n');
```

Comment:
I don't know if it's clear enough for the users to only mention the name of the user. It can be better to put `Message de : @Ayoub` or something else 🤷‍♂️